kong/plugins/cors/handler.lua

```diff
@@ -56,8 +58,25 @@ end
 
 local function configure_credentials(ngx, conf)
   if ngx.ctx.cors_allow_all then
-    ngx.header["Access-Control-Allow-Credentials"] = "false"
+    local allow_credentials = false
+
+    -- If it's catch all, then match the Origin requested by the client
+    -- but only if the Origin URL is valid, and only if credentials=true
```

Comment:
Is there any part of the specification that requires us to only set the ACAO if it is a valid URL?